interface/src/app/analyze/analysis/analysis.js

```diff
@@ -29,9 +29,9 @@ angular.module('adage.analyze.analysis', [
   });
 }])
 
-.controller('AnalysisCtrl', ['$scope', '$log', '$location', '$q', 'Sample',
-'Activity', 'AnnotationType', 'SampleBin',
-function AnalysisCtrl($scope, $log, $location, $q, Sample, Activity,
+.controller('AnalysisCtrl', ['$scope', '$log', '$location', '$q', '$state',
+'Sample', 'Activity', 'AnnotationType', 'SampleBin',
+function AnalysisCtrl($scope, $log, $location, $q, $state, Sample, Activity,
```

Comment:
Good catch. I will fix that!

- [ ] check dependencies injected into `AnalysisCtrl`